examples/csv_deployment/sample_deployment.csv

```diff
@@ -2,6 +2,17 @@ MetroAE deployment template CSV,
 This Spreadsheet can be used as an Input to MetroAE,
 ,
 ,,
+,Nuage Utility Host VM,
```

Comment:
Should there be changes to this file?

Reply:
Generate all from schema changed this file. Maybe there was a change from before which changed the schemas but did not generate example files. I haven't made any manual changes

Reply:
Merged with the latest dev, now we don't see the diff